src/main/webapp/chrome-extension/popup.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="popup.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <main>
+      <nav>
+        <span class="bold">starfish</span>
+        <img src="images/starfish_512.png">
+      </nav>
+      <section id="user-info">
+        <span class="bold heading">User</span>
+        <div class="card">
+          <span id="email"></span>
+        </card>
+      </section>
+      <section id="gnote-form">
+        <span class="bold heading">gNote</span>
+        <div class="card" id="gnote-card">
+          <span>Title: <input type="text" id="doc-name-input"></span>
+          <span><input type="checkbox"> Share Publicly</span>
+          <button id="generate-note-btn">Generate</button>
+          <div id="loading">
+            <img src="images/loading.gif">
+          </div>
+        </div>
+      </section>
+      <section id="footer">
```

Comment:
This isn't a big deal, but since you used the "nav" semantic tag, it's worth noting that there's also a "footer" semantic tag. Would be a good idea to remain consistent and use the semantic tag instead of giving an id="footer".